exercise/templates/exercise/_points_badge.html

```diff
@@ -6,27 +6,34 @@
 </span>
 
 {% elif confirm_the_level %}
-{% if submitted %}
-<span class="badge{% if graded and official %} badge-{% if full_score %}success{% elif passed %}warning{% else %}danger{% endif %}{% endif %} {{ classes }}">
-  {% if not official %}{% trans 'Submitted after deadline' %}{% elif passed %}{% trans "Accepted" %}{% elif graded %}{% trans "Rejected" %}{% else %}{% trans "Submitted" %}{% endif %}
-</span>
-{% elif confirmable_points %}
-<span class="badge badge-danger" data-toggle="tooltip" title="{% trans 'You must submit here to confirm your points. Submissions are accepted even after the deadline.' %}">
-  {% trans "Missing" %}
-</span>
-{% elif exercise_page %}
-<span class="badge">
-  {% trans "Not submitted" %}
-</span>
-{% endif %}
+	{% if submitted %}
+	<span class="badge{% if graded and official %} badge-{% if full_score %}success{% elif passed %}warning{% else %}danger{% endif %}{% endif %} {{ classes }}">
+	  {% if not official %}{% trans 'Submitted after deadline' %}{% elif passed %}{% trans "Accepted" %}{% elif graded %}{% trans "Rejected" %}{% else %}{% trans "Submitted" %}{% endif %}
+	</span>
+	{% elif confirmable_points %}
+	<span class="badge badge-danger" data-toggle="tooltip" title="{% trans 'You must submit here to confirm your points. Submissions are accepted even after the deadline.' %}">
+	  {% trans "Missing" %}
+	</span>
+	{% elif exercise_page %}
+	<span class="badge">
+	  {% trans "Not submitted" %}
+	</span>
+	{% endif %}
 
 {% else %}
-{% if difficulty %}
-<span class="difficulty">{{ difficulty }}</span>
-{% endif %}
-<span class="badge{% if submitted and official %} {% if full_score %}badge-success{% elif passed %}badge-warning{% else %}badge-danger{% endif %}{% if unconfirmed %} unconfirmed-points{% endif %}{% endif %}{% if classes %} {{ classes }}{% endif %}"
+	{% if difficulty and difficulty != "-" %}
+	<span class="difficulty">{{ difficulty }}</span>
+	{% endif %}
+<span class="badge{% if submitted and official %} {% if full_score %}badge-success{% elif passed %}badge-warning{% else %}badge-danger{% endif %}{% if unconfirmed %} unconfirmed-points{% endif %}{% endif %}{% if classes %} {{ classes }}{% endif %}
+            {% if difficulty == "-" and max <= 2 %} aplus-points
+            {% if points == 1 %}{% if max == 2 %}ok{% elif max == 1%}accepted{% endif %}{% elif points == 2 %}accepted{% elif points == 0 and submitted %}rejected{% endif %}
```

Comment:
This could be a bit more clear as a templatetag. do the logic in python and just get the class out. Thus, we can render the default text with help of the same function below. For example:
`{% get_points_class %}`